itest/full_value_split_test.go

```diff
@@ -0,0 +1,76 @@
+package itest
+
+import (
+	"bytes"
+	"context"
+
+	"github.com/lightninglabs/taro/chanutils"
+	"github.com/lightninglabs/taro/tarorpc"
+	"github.com/lightningnetwork/lnd/lntest/wait"
+	"github.com/stretchr/testify/require"
+)
+
+// testFullValueSplitSend tests that we can properly send the full value of a
+// normal asset.
+func testFullValueSend(t *harnessTest) {
+	// First, we'll make an normal assets with enough units to allow us to
+	// send it around a few times.
+	rpcAssets := mintAssetsConfirmBatch(
+		t, t.tarod, []*tarorpc.MintAssetRequest{simpleAssets[0]},
+	)
+
+	genBootstrap := rpcAssets[0].AssetGenesis.GenesisBootstrapInfo
+
+	ctxb := context.Background()
+
+	// Now that we have the asset created, we'll make a new node that'll
+	// serve as the node which'll receive the assets.
+	secondTarod := setupTarodHarness(
+		t.t, t, t.lndHarness.BackendCfg, t.lndHarness.Bob, t.universeServer,
+	)
+	defer func() {
+		require.NoError(t.t, secondTarod.stop(true))
+	}()
+
+	var (
+		numSends   = 1
+		fullAmount = rpcAssets[0].Amount
+	)
+
+	// 3 sends, to check for DB issues?
```

Comment:
Yeah can't hurt to ping pong it back. 

We can also confirm that the zero value leaves don't mess anything up for repeated sends.